client/dashboard/sites/settings-sftp-ssh/summary.tsx

```diff
@@ -0,0 +1,45 @@
+import { useQuery } from '@tanstack/react-query';
+import { Icon } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { file } from '@wordpress/icons';
+import { siteSftpUsersQuery, siteSshAccessStatusQuery } from '../../app/queries';
+import RouterLinkSummaryButton from '../../components/router-link-summary-button';
+import { canUseSftp, canUseSsh } from '../../utils/site-features';
+import type { Site } from '../../data/types';
+
+export default function SftpSshSettingsSummary( { site }: { site: Site } ) {
+	const { data: sftpUsers } = useQuery( {
+		...siteSftpUsersQuery( site.slug ),
+		enabled: canUseSftp( site ),
+	} );
+
+	const { data: sshAccessStatus } = useQuery( {
+		...siteSshAccessStatusQuery( site.slug ),
+		enabled: canUseSsh( site ),
+	} );
+
+	const sftpEnabled = sftpUsers && sftpUsers.length > 0;
+
+	const sshEnabled = sshAccessStatus?.setting === 'ssh';
+
+	const badges = [
+		{
+			text: sftpEnabled ? __( 'SFTP enabled' ) : __( 'SFTP disabled' ),
+			intent: sftpEnabled ? ( 'success' as const ) : undefined,
+		},
+		canUseSsh( site ) && {
+			text: sshEnabled ? __( 'SSH enabled' ) : __( 'SSH disabled' ),
+			intent: sshEnabled ? ( 'success' as const ) : undefined,
+		},
+	].filter( ( badge ) => !! badge );
+
+	return (
+		<RouterLinkSummaryButton
+			to={ `/sites/${ site.slug }/settings/sftp-ssh` }
+			title="SFTP/SSH"
+			density="medium"
```

Comment:
We now need to pass `density` from the parent `SummaryButtonList`; please check the other setting summaries on `trunk`.